src/hnsw/insert.c

```diff
@@ -98,6 +98,7 @@ bool ldb_aminsert(Relation         index,
         hdr_page = BufferGetPage(hdr_buf);
         hdr = (HnswIndexHeaderPage *)PageGetContents(hdr_page);
         assert(hdr->magicNumber == LDB_WAL_MAGIC_NUMBER);
+        HEADER_FOR_EXTERNAL_RETRIEVER = *hdr;
```

Comment:
no need to add here, as it's assigned later out of if block

Reply:
needed because `ldb_wal_retriever_area_init` uses it :)